portal-frontend/src/app/shared/button-container/button-container.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="button-container">
+  <button *ngIf="!draftMode" (click)="onSaveExit()" mat-stroked-button color="accent">Save and Exit</button>
```

Comment:
Since these buttons are identical besides text, consider changing to:

```html
<button (click)="onSaveExit()" mat-stroked-button color="accent">{{ draftMode ? 'Discard all Changes' : 'Save and Exit'</button>
```

Reply:
gotcha, made changes